docs_user/modules/proc_migrating-mon-from-controller-nodes.adoc

```diff
@@ -0,0 +1,411 @@
+[id="migrating-mon-from-controller-nodes_{context}"]
+
+= Migrating Ceph Monitor daemons to {Ceph} nodes
+
+The following section describes how to move Ceph Monitor daemons from the
+OpenStack controller nodes to a set of target nodes. Target nodes might be
+pre-existing {Ceph} nodes, or OpenStack Compute nodes if Ceph is deployed by
+{OpenStackPreviousInstaller} with an HCI topology.
+
+.Prerequisites
+
+Configure the target nodes (CephStorage or ComputeHCI) to have both `storage`
+and `storage_mgmt` networks to ensure that you can use both {Ceph} public and
+cluster networks from the same node. This step requires you to interact with
+{OpenStackPreviousInstaller}. From {rhos_prev_long} {rhos_prev_ver} and later
+you do not have to run a stack update. However, there are commands that you
+must perform to run `os-net-config` on the bare metal node and configure
+additional networks.
+
+.. If target nodes are `CephStorage`, ensure that the network is defined in the
+`metalsmith.yaml` for the CephStorageNodes:
++
+[source,yaml]
+----
+  - name: CephStorage
+    count: 2
+    instances:
+      - hostname: oc0-ceph-0
+        name: oc0-ceph-0
+      - hostname: oc0-ceph-1
+        name: oc0-ceph-1
+    defaults:
+      networks:
+        - network: ctlplane
+          vif: true
+        - network: storage_cloud_0
+            subnet: storage_cloud_0_subnet
+        - network: storage_mgmt_cloud_0
+            subnet: storage_mgmt_cloud_0_subnet
+      network_config:
+        template: templates/single_nic_vlans/single_nic_vlans_storage.j2
+----
+
+.. Run the following command:
++
+----
+openstack overcloud node provision \
+  -o overcloud-baremetal-deployed-0.yaml --stack overcloud-0 \
+  --network-config -y --concurrency 2 /home/stack/metalsmith-0.yam
+----
+
+.. Verify that the storage network is configured on the target nodes:
++
+----
+(undercloud) [stack@undercloud ~]$ ssh heat-admin@192.168.24.14 ip -o -4 a
+1: lo    inet 127.0.0.1/8 scope host lo\       valid_lft forever preferred_lft forever
+5: br-storage    inet 192.168.24.14/24 brd 192.168.24.255 scope global br-storage\       valid_lft forever preferred_lft forever
+6: vlan1    inet 192.168.24.14/24 brd 192.168.24.255 scope global vlan1\       valid_lft forever preferred_lft forever
+7: vlan11    inet 172.16.11.172/24 brd 172.16.11.255 scope global vlan11\       valid_lft forever preferred_lft forever
+8: vlan12    inet 172.16.12.46/24 brd 172.16.12.255 scope global vlan12\       valid_lft forever preferred_lft forever
+----
+
+.Procedure
+
+This procedure assumes that some of the steps are run on the source node that
+we want to decommission, while other steps are run on the target node that is
+supposed to host the redeployed daemon.
```

Comment:
Move this paragraph after "... or OpenStack Compute nodes if Ceph is deployed by director with an HCI topology."

Reply:
+1

Reply:
Done